.github/workflows/build.yml

```diff
@@ -31,24 +32,24 @@ jobs:
         python-version: ["3.8", "3.9", "3.10", "3.11"]
     steps:
       - uses: actions/checkout@v3
+      - uses: pdm-project/setup-pdm@v3
       - name: Set up Python ${{ matrix.python-version }}
         uses: actions/setup-python@v4
         with:
           python-version: ${{ matrix.python-version }}
       - name: Install packages and dependencies
         run: |
           python -m pip install --upgrade pip wheel
-          pip install -e .
+          pdm install -d
```

Comment:
why this change? Pip install should work as before.

Reply:
see the comment above.

Apart from that, you should not use `-d` here as it will install `dev` dependencies which are not needed here.